projects/1.18-fabric/assets/tooltiptweaks/tooltiptweaks/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+    "tooltiptweaks.modrinth": "Modrinth",
+    "tooltiptweaks.curseforge": "CurseForge",
+    
+    "tooltiptweaks.menu.options": "Tooltip Tweaks 配置",
+
+    "tooltiptweaks.options.durability.display": "显示耐久度",
+    "tooltiptweaks.options.durability.percentage.decimals": "耐久度百分比",
+    "tooltiptweaks.options.durability.remaininguses": "显示 \"剩余耐久\"",
+    "tooltiptweaks.options.durability.remaininguses.threshold": "剩余耐久显示阈值",
+
+    "tooltiptweaks.options.food.effects": "显示食物药水效果",
+    "tooltiptweaks.options.food.hunger": "显示食物的饱食度回复",
+    "tooltiptweaks.options.food.negative": "隐藏负面食物的药水效果",
+    "tooltiptweaks.options.food.saturation": "显示食物的饱和度回复",
+    "tooltiptweaks.options.food.stew": "隐藏谜之炖菜的药水效果",
+    "tooltiptweaks.options.food.stew.negative": "隐藏谜之炖菜的负面药水效果",
+
+    "tooltiptweaks.options.tools.beehive.bees": "显示蜂巢蜜蜂数",
+    "tooltiptweaks.options.tools.beehive.honey": "显示蜂巢蜂蜜量",
+    "tooltiptweaks.options.tools.clock.display": "显示时钟时间",
+    "tooltiptweaks.options.tools.compass.lodestone": "显示 \"指南针所绑定的磁石\" 的坐标",
+    "tooltiptweaks.options.tools.shulker_box.display": "显示潜影盒内容",
+    "tooltiptweaks.options.tools.shulker_box.entries": "潜影盒预览物品行数",
+
+    "tooltiptweaks.options.value.12hour": "§a12小时制",
+    "tooltiptweaks.options.value.24hour": "§a24小时制",
+    "tooltiptweaks.options.value.advanced_tooltips": "§e仅限高级提示框",
+    "tooltiptweaks.options.value.fraction": "§a数值",
+    "tooltiptweaks.options.value.low_durability": "§e低耐久时",
+    "tooltiptweaks.options.value.off": "§c否",
+    "tooltiptweaks.options.value.on": "§a是",
+    "tooltiptweaks.options.value.percentage": "§a百分比",
+    "tooltiptweaks.options.value.shulker_box.vanilla": "§a原版（显示每格物品）",
+    "tooltiptweaks.options.value.shulker_box.enhanced": "§a增强（显示物品总数）",
+    "tooltiptweaks.options.value.shulker_box.inventory": "§a详细库存",
+    "tooltiptweaks.options.value.shulker_box.compact": "紧凑",
+
+    "tooltiptweaks.section.durability": "耐久",
+    "tooltiptweaks.section.food": "食物",
+    "tooltiptweaks.section.tools": "其他",
+
+    "tooltiptweaks.ui.beehive.bees": "内有%s蜜蜂",
+    "tooltiptweaks.ui.beehive.honey": "内有%s蜂蜜",
+    "tooltiptweaks.ui.clock.unknown": "？？？",
+    "tooltiptweaks.ui.lodestone_compass.dimension": "维度：\"%s\"",
+    "tooltiptweaks.ui.lodestone_compass.position": "坐标：\"%s, %s, %s\"",
```

```suggestion
    "tooltiptweaks.ui.lodestone_compass.dimension": "维度：“%s”",
    "tooltiptweaks.ui.lodestone_compass.position": "坐标：“%s, %s, %s”",
```
坐标里面的逗号不改比较好，但改了也没事
这个引号如果里面是ID什么的就不改了